### ISSUE TITLE
Why not pretrain yolov5 with ImageNet data?

### ISSUE THREAD

**Issue by markncx** [ultralytics/yolov5#951]
## ❔Question

I have a question. yolo-v1, v2, v3, v4 all pretrain with imagenet, and then, pretrain with MS COCO. However, yolo-v5 pretrains with MS COCO (without imagenet). Can pretraining with imagenet improve the AP of yolo-v5?

**Comment by glenn-jocher** [comment]
@markncx no pretraining of any kind is used for YOLOv5. All COCO results shown in the README are achieved by training YOLOv5 from scratch.

If you find positive results through alternative initialization techniques please advise us!

**Comment by MorphSeur** [comment]
Hello, is any kind of trained Yolo v5 model on ImageNet dataset since?

**Comment by glenn-jocher** [comment]
@MorphSeur no pretraining of any kind is used for YOLOv5. All COCO results shown in the README are achieved by training YOLOv5 from scratch.

If you find positive results through alternative initialization techniques please advise us!

**Comment by zhiqwang** [comment]
FYI @MorphSeur and @glenn-jocher ,

The repository https://github.com/gau-nernst/vision-toolbox#darknet uses YOLOv5's DarkNet to train a classification model on ImageNet, or you can use it as a reference:

This implementation has the following differences. (checkout this https://github.com/gau-nernst/vision-toolbox/issues/2 for more details)
> The classification performance doesn't look so good compared to CSPDarknet-53, probably because of my deviations from the original YOLOv5 Darknet (ReLU vs SiLU, Batch norm parameters). YOLOv5 backbone is also tuned for object detection, so its classification performance may not be optimal. Or maybe there are just noises in the training process.

cc @gau-nernst
